common/models/facility.js

```diff
@@ -454,17 +445,11 @@ module.exports = (Facility) => {
 
 
   Facility.facilitiesByTypeAndOwnership = async (districts, cb) => {
-    const IDs = await getDistrictsIDs(districts);
-
-    let facilities = null;
-    if (!IDs.length) {
-      facilities = await server.models.Facility.find();
-    } else {
-      facilities = await server.models.Facility.find({ where: { district_id: { inq: IDs } } });
-    }
-    const owners = await server.models.Owner.find();
-    const facilityTypes = await server.models.FacilityType.find();
-
+    const IDs = await getDistrictsIDs(districts)
+    const where = IDs.length ? { district_id: { inq: IDs } } : {}
+    const facilities = await Facility.find({ where })
+    const owners = await server.models.Owner.find()
+    const facilityTypes = await server.models.FacilityType.find()
     const mapped  = [];
```

Comment:
This should not be an empty array. Also, the name must be more descriptive of what it's returning, i.e. MappedOwners

```mapped = owners.map(owner => {
    return {
        ...owner,
       types: facilityTypes
    }
})```